script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt

```diff
@@ -55,20 +55,20 @@ With a sudden return to normal life, I start to breathe properly again.
 Just as she says her farewells, the girl asks me a strange question.
 <Page17>
 "...Well, I was born with it, I guess.
-Ever since I was a kid, I get dizzy whenever I exercise a little."
+Ever since I was a kid, if I exercise even a little I'll end up getting dizzy."
 "Hmm. I guess dogs are out of the question then. You can't play with them after all."
-"No, I don't have a problem going for a walk."
+"No, just going for a walk isn't really a problem."
 I don't have any desire to get a dog, but it's kind of sad to completely deny it.
 <Page18>
 "A walk....
 Your face is still pale. I suggest that you go home quickly and get some rest."
 <Page19>
 The girl walks away.
 "Huh...?"
-The moment she left, for some reason, it felt like her condition was many times more worse than mine.
+As she left, I somehow got the feeling that her own physical condition was in far worse shape than mine.
```

Comment:
I think it should be either 'I somehow got the feeling that her own physical condition was far worse than mine' or 'I somehow got the feeling that she was in far worse shape than me.'